apps/trustlab/src/components/Page/Page.js

```diff
@@ -0,0 +1,16 @@
+import { NextSeo } from "next-seo";
+import React from "react";
+
+function Page({ children, seo }) {
+  return (
+    <>
+      <NextSeo {...seo} />
+      <div>
+        {/* TODO: Add Navbar and footer */}
+        {children ? <main>{children}</main> : null}
+      </div>
```

Comment:
Why not just:
```suggestion
      {/* TODO: Add Navbar and footer */}
      {children ? <main>{children}</main> : null}
```
i.e., do we need the `<div>`?